sbi/user_input/user_input_checks.py

```diff
@@ -208,12 +208,13 @@ def check_for_possibly_batched_observations(x_o: Tensor):
     additional leading batch dimension of size 1.
     """
 
+    x_ndim = len(x_shape)
```

Comment:
num_xdim?

Reply:
I would like to have `ndim` in there since it is most precise imo.